utils/chain-ops/genesis/layer_one.go

```diff
@@ -333,14 +352,21 @@ func deployL1Contracts(config *DeployConfig, backend *backends.SimulatedBackend)
 				defaultResourceConfig,
 			},
 		},
+		{
+			Name: "ValidatorPool",
+			Args: []interface{}{
+				config.ValidatorPoolTrustedValidator,
+				config.ValidatorPoolMinBondAmount.ToInt(),
+			},
+		},
 		{
 			Name: "L2OutputOracle",
 			Args: []interface{}{
 				uint642Big(config.L2OutputOracleSubmissionInterval),
 				uint642Big(config.L2BlockTime),
 				big.NewInt(0),
 				uint642Big(uint64(config.L1GenesisBlockTimestamp)),
-				config.L2OutputOracleValidator,
+				predeploys.DevValidatorPoolAddr,
```

Comment:
just a question, why some of predeployed dev L1 contracts addresses are added here, and some other ones are added below `l1Deployer` like [line 450](https://github.com/kroma-network/kroma/pull/56/files#diff-218d848c5297513155401cc5da24b75691a0d263df3c84d99a11bf9913374d95R450)?

Reply:
This is because dev L1 contract address remains the same in the production environment as well.

Reply:
My guess is that l1Deployer passes it directly if the predeployed address is initialized, and deployL1Contracts passes it otherwise. 

In my opinion, if the predeployed addresses are at the beginning of the parameter, these are added in the `l1Deployer`. Otherwise, added in `deployL1Contracts`

Reply:
In my opinion, if the predeployed addresses are at the beginning of the parameter, these are added in the l1Deployer. Otherwise, added in deployL1Contracts

Reply:
if there is no reasonable reason, I think it would be better to unify the position of the predeployed address additions between `deployL1Contracts` and `l1Deployer`.